src/test/java/core/basesyntax/service/WriterServiceImplTest.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.impl.WriterServiceImpl;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import org.junit.Test;
+
+public class WriterServiceImplTest {
+    private static final String DIRECTORY = "src/test/resources";
+    private static final String FILE_NAME_ACTUAL_REPORT = "src/test/resources/actualReport.csv";
+    private static final String REPORT = "fruit,quantity"
+            + System.lineSeparator()
+            + "banana,152"
+            + System.lineSeparator()
+            + "apple,90";
+    private static final WriterServiceImpl writerService = new WriterServiceImpl();
```

Comment:
you should initialize WriterServiceImpl in the method with BeforeClass annotation